core/src/main/scala/kafka/log/LogSegment.scala

```diff
@@ -444,3 +451,7 @@ class LogSegment(val log: FileMessageSet,
     timeIndex.file.setLastModified(ms)
   }
 }
+
+object LogFlushStats extends KafkaMetricsGroup {
```

Comment:
It's used in this file. It may not show in the diff because I moved it from `FileMessageSet` (which was deleted).